average.js

```diff
@@ -1,6 +1,9 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  //filter NaN
+  const filteredNumbers = numbers.filter(function (num) {
```

Comment:
Good understanding of the test and translation into a filter function :+1: 
It could be further concise using an arrow function `(num)=> ...`
You could try it out.